src/styles/blocks/header.scss

```diff
@@ -0,0 +1,59 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo-link {
+  display: flex;
+  cursor: pointer;
+
+  @include transition;
```

Comment:
It seems like you are using an undefined mixin `transition`. Make sure that you have defined it in your `mixins.scss` file. If it is not defined, the SCSS will not compile correctly. Define a mixin for transition if it's a repetitive style, or apply the transition properties directly here.